scripts/train_lm.py

```diff
@@ -59,12 +67,17 @@
                     'It is valid only when --cleanup=true')
 parser.add_argument("--max-memory", type=str, default='',
                     help="Memory limitation for sort called by get_counts.py.")
+parser.add_argument("--lm-dir", type=str, default='',
+                    help="Output directory where the language model is created."
+                    "If this is not specified, the output directory would be a subdirectory under <work-dir>, "
+                    "with name '<vocab_name>_<order>.pocolm', where the <vocab_name> will be the name of wordlist "
+                    "if --wordlist is specified otherwise the size of vocabulary, and <order> is the ngram order of model.")
```

Comment:
It complains that 'TypeError: 'required' is an invalid argument for positionals'. And set it default to '' would not work either. I'm trying to goole some solutions.